packages/core/src/version.ts

```diff
@@ -0,0 +1 @@
+export const VERSION = '0.0.0';
```

Comment:
```suggestion
export const VERSION = '0.0.0-dev';
```

Reply:
this is generated file

Reply:
We can use package.json for this. It is a json file after all